### ISSUE TITLE
Remove right-justified colons between labels and fields on forms

### ISSUE THREAD

**Issue by brainwane** [SolutionGuidance/psm#376]
On forms like the provider enrollment form, the colons on the labels are right-justified and it looks disconnected.

See https://github.com/OpenTechStrategies/psm/issues/308#issue-246414550  for a screenshot.

**Comment by jasonaowen** [comment]
I noticed those, too. I propose deleting them - I don't think they're adding any meaningful signal.

**Comment by cecilia-donnelly** [comment]
@jvasile writes: "Thanks, Sumana. Perhaps we should just remove the colons entirely. The help icon followed by a colon might look weird." [on psm-dev](https://groups.google.com/forum/#!topic/psm-dev/xX-WQjd-UEc).

I'm in favor of removing them, too!

**Comment by brainwane** [comment]
This is also something we should do for Section 508 compliance; `pa11y`'s `Section508.D.H49.B` warning tells us, "Semantic markup should be used to mark emphasised or special text so that it can be programmatically determined." So `<b>:</b>` is also poor practice for accessibility.

**Comment by cecilia-donnelly** [comment]
@PaulMorris  suggests that we could remove these and right-align the labels instead.

**Comment by PaulMorris** [comment]
Most places removing the colons should be a nice improvement.  In a few places just taking them out will decrease clarity, like on the advanced search page (which is already not very clear about what goes with what):

![screenshot-2018-2-28 advanced search 2](https://user-images.githubusercontent.com/1091693/36817218-5bc970c4-1cae-11e8-8130-0654d3d4cc5d.png)

Some way of grouping elements would help in places like this.

**Comment by kfogel** [comment]
This may be related to issue #1045, which is about similarly oddly-aligned colons in PDF exports.
